### PR TITLE
Update image

### PR DESCRIPTION
**Pull request by naseemkullah**
Thought it would be good to update the default value of image version as many improvements have been made since 0.16.0. 

https://github.com/oliver006/redis_exporter/releases

### CODE REVIEW

File: stable/prometheus-redis-exporter/values.yaml [hunk 1]
```diff
@@ -13,7 +13,7 @@ serviceAccount:
 replicaCount: 1
 image:
   repository: oliver006/redis_exporter
```
Comment (davidkarlsen) [inline stable/prometheus-redis-exporter/values.yaml:15]: Please update the readme too.
TIA

Comment (naseemkullah) [reply stable/prometheus-redis-exporter/values.yaml:15]: Oops, thanks.
Done!

**Pull request merged**